commit 88a39c1c44632be6287a8f95b3cbf15cb5120719
Author: lorenz <looooo@users.noreply.github.com>
Date:   Tue Dec 27 04:59:35 2022 +0100

    FEM: femmesh: fix AttributeError: module 'numpy' has no attribute 'int'

diff --git a/src/Mod/Fem/femmesh/meshtools.py b/src/Mod/Fem/femmesh/meshtools.py
index d09b187fcd..fdcdb7e825 100644
--- a/src/Mod/Fem/femmesh/meshtools.py
+++ b/src/Mod/Fem/femmesh/meshtools.py
@@ -489,7 +489,7 @@ def get_femelement_sets(
     count_femelements = 0
     referenced_femelements = np.zeros(
         (max(femelement_table.keys()) + 1,),
-        dtype=np.int
+        dtype=int
     )
     has_remaining_femelements = None
     for fem_object_i, fem_object in enumerate(fem_objects):
